pdr_backend/cli/cli_module_lake.py

```diff
@@ -63,7 +64,7 @@ def do_lake_query(args, ppss):
 @enforce_types
 def do_lake_raw_drop(args, ppss):
     pds = PersistentDataStore(ppss.lake_ss.lake_dir, read_only=False)
-    drop_tables_from_st(pds, "raw", args.ST)
+    drop_tables_from_st(pds, "raw", UnixTimeMs(args.ST))
```

Comment:
please remember to remove this conversion here, as well. See comments in #1080

Reply:
Done, please review.